app/views/hyrax/admin/stats/show.html.erb

```diff
@@ -1,21 +1,29 @@
 <% content_for :page_header do %>
   <h1><span class="fa fa-edit"></span> Statistics for <%= application_name %></h1>
 <% end %>
-  
-  <div class="panel panel-default tabs">
+
+  <div class="card tabs">
     <!-- Nav tabs -->
     <ul id="statistics-tabs" class="nav nav-tabs" role="tablist">
-      <li role="presentation" class="active"><a href="#collections" aria-controls="collections" role="tab" data-toggle="tab">Collections</a></li>
```

Comment:
🟡  I'm a trepidatious about removing a `role="presentation"` as that might have been added via an accessibility audit.

We'll want to look at the commit that introduced `role="presentation"` and see if there's any language about it.  It's also possible that this is something to sleuth in Hyrax.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role

Reply:
I can definitely bring this back, but from the bootstrap examples, i see that they include the role in bootstrap 3
https://getbootstrap.com/docs/3.3/components/#nav-tabs

but did not in bootstrap 4 
https://getbootstrap.com/docs/4.0/components/navs/#tabs

I'm not sure if originally it was implemented simply because they were following the documentation

Reply:
Let's not worry about it.